.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             - name: 'Clean tree'
               run: 'npm run test-clean-tree'
     integration:
-        runs-on: ubuntu-latest
+        runs-on: macos-14
```

Comment:
Do we need to make this change? The cost of the runners are significantly different iirc.

Reply:
I was having troubles with the Linux box taking over 5 min to install playwright, probably a temporary thing.

But then I thought it would be useful to match what the majority of devs would be using. Are we currently paying then? I thought being open source mean's it's free? 

I can switch it back if needed,